src/privileged/autoplay/api.js

```diff
@@ -0,0 +1,397 @@
+"use strict";
+
+ChromeUtils.import("resource:///modules/PermissionUI.jsm");
+ChromeUtils.import("resource:///modules/SitePermissions.jsm");
+ChromeUtils.import("resource://gre/modules/ExtensionCommon.jsm");
+ChromeUtils.import("resource://gre/modules/ExtensionStorage.jsm");
+ChromeUtils.import("resource://gre/modules/PopupNotifications.jsm");
+ChromeUtils.import("resource://gre/modules/PrivateBrowsingUtils.jsm");
+ChromeUtils.import("resource://gre/modules/Services.jsm");
+ChromeUtils.import("resource://gre/modules/TelemetryController.jsm");
+
+const CID = ChromeUtils.import("resource://gre/modules/ClientID.jsm", {});
+const { AddonStudies } = ChromeUtils.import("resource://normandy/lib/AddonStudies.jsm", {});
+const { EventManager } = ExtensionCommon;
+
+XPCOMUtils.defineLazyGetter(this, "gBrowserBundle", function() {
```

Comment:
You don't need to import `XPCOMUtils` for this anymore, `ChromeUtils` supports it like this:
```
ChromeUtils.defineModuleGetter(this, "MyModule",
                               "resource://MyModule.jsm");
```

Reply:
But here I don't define a module, I defined a getter for the variable. It seems not equal to the usage of `defineModuleGetter()`?